pkgs/development/ocaml-modules/ocplib-json-typed/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchgit, ocaml, findlib, ocplib-endian, uri }:
+
+let version = "0.4"; in
+
+stdenv.mkDerivation {
+  name = "ocaml-ocplib-json-typed-${version}";
+
+  src = fetchgit {
```

Comment:
You should use `fetchFromGitHub` instead of fetchgit (if the repo doesn't contain submodules)